src/core/core.scale.js

```diff
@@ -138,7 +140,9 @@ function parseFontOptions(options, nestedOpts) {
 		fontFamily: valueOrDefault(nestedOpts.fontFamily, options.fontFamily),
 		fontSize: valueOrDefault(nestedOpts.fontSize, options.fontSize),
 		fontStyle: valueOrDefault(nestedOpts.fontStyle, options.fontStyle),
-		lineHeight: valueOrDefault(nestedOpts.lineHeight, options.lineHeight)
+		lineHeight: valueOrDefault(nestedOpts.lineHeight, options.lineHeight),
+		lineWidth: valueOrDefault(nestedOpts.lineWidth, options.lineWidth),
```

Comment:
These need to go down with `color`, then the `tickFont` wil have those.

Reply:
That was the key!!!! 🎉 🎉 

![image](https://user-images.githubusercontent.com/676681/72179740-e1f9e700-33e5-11ea-8991-9f39bd947ae7.png)

```
ticks: {
    major: {
        enabled: true,
        fontStyle: 'bold',
	lineWidth: 3,
	strokeStyle: 'blue',
	fontColor: 'white',
    },
    minor: {
        enabled: true,
	fontStyle: 'bold',
	lineWidth: 1.8,
	strokeStyle: 'red',
	fontColor: 'white',
    },
```
Thanks @kurkle !